MapleServer2/Types/Item.cs

```diff
@@ -308,5 +310,16 @@ public static Item ShoesMale()
                 Stats = new ItemStats(),
             };
         }
+
+        public static Item CreateNew(Item other)
+        {
+            Item item = new Item(other)
+            {
+                Amount = 1,
+                Slot = -1,
+                Uid = GuidGenerator.Long()
+            };
+            return item;
+        }
```

Comment:
This seems like an unnecessary addition to item since we're just setting 3 attributes, better to do this in the controller I think.